package.json

```diff
@@ -55,7 +56,8 @@
     "hoist-non-react-statics": "^1.2.0",
     "immutable": "^3.8.1",
     "jwt-decode": "^2.2.0",
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
+    "opencollective": "^1.0.3"
```

Comment:
We tried that at the beginning but it was breaking because the post install scripts run in all environments. The `opencollective postinstall` script will detect the environment and only run in dev though.